config/locales/nl.yml

```diff
@@ -116,16 +116,16 @@ nl:
     index:
       subtitle: 'Het leven is een rommeltje, categoriseer mensen, plaatsen en dingen.'
       premade1_name: Familie
-      premade1_description: 'Familie gerelateerd'
+      premade1_description: 'Gerelateerd met familie'
       premade2_name: Vrienden
-      premade2_description: 'Vrienden gerelateerd'
+      premade2_description: 'Gerelateerd met vrienden'
```

Comment:
Gerelateerd aan ipv met?

Reply:
net ontdekt dat ik altijd "gerelateerd met" zeg terwijl het inderdaad "gerelateerd aan" is, heb ik ook weer iets bijgeleerd ;)